<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="6,2">
       <item>
        <widget class="QTextBrowser" name="text_receive"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>端口</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmb_port"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmb_rate">
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>57600</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmb_data">
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>停止位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmb_stop">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>校验位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmb_check">
             <item>
              <property name="text">
               <string>无</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>奇</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>偶</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_open">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>打开串口</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cleanR">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>清空接收区</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="6,2">
       <item>
        <widget class="QTextEdit" name="text_send"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,3,1,1">
         <item>
          <widget class="QPushButton" name="btn_send">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cleanS">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>清空发送区</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>自动发送周期</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="text_ms">
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QCheckBox" name="cb_newLine">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>    发送新行</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_auto">
             <property name="text">
              <string>自动发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
